js_modules/dagit/packages/ui/src/components/Suggest.tsx

```diff
@@ -55,7 +55,7 @@ export const Suggest = <T,>(props: React.PropsWithChildren<SuggestProps<T>>) =>
     ...props.popoverProps,
     minimal: true,
     modifiers: deepmerge(
-      {offset: {enabled: true, offset: '0, 8px'}},
+      {offset: {enabled: true, offset: '-5px 8px'}},
```

Comment:
Is this always correct for all `Suggest`?

Reply:
Hmm, actually I compared this to a few of our other dropdowns like the "runs filter", and it looks like the -5px is aligning the titles of the dropdown options with the text in the box but is not consistent with what we do in the run filter... I'll go ahead and revert this change for now and we can assess whether everything should be bumped out 5px so the text aligns.